explainaboard/analysis/case.py

```diff
@@ -80,6 +81,36 @@ class AnalysisCaseLabeledSpan(AnalysisCaseSpan):
     predicted_label: str
 
 
+@dataclass
+class AnalysisCaseBlock(AnalysisCase):
+
+    text: str
+    orig_str: str
```

Comment:
This class isn't used anywhere other than in `AnalysisCaseLabeledBlock`, so could we delete it and move the members into the below class?
```suggestion
```